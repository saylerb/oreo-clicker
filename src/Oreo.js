import React, { Fragment } from "react";
import SVGInline from "react-svg-inline";
import oreo from "./oreo.svg";

const Oreo = () => (
  <Fragment>
    <SVGInline className="oreo" classSuffix="-svg" width="300" svg={oreo} />
  </Fragment>
);

export default Oreo;
