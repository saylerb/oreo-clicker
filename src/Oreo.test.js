import React from "react";
import { create } from "react-test-renderer";
import Oreo from "./Oreo";

test("snapshot", () => {
  const c = create(<Oreo />);
  expect(c.toJSON()).toMatchSnapshot();
});
