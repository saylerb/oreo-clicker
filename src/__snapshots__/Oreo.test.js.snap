// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot 1`] = `
<span
  className="SVGInline oreo"
  dangerouslySetInnerHTML={
    Object {
      "__html": "fake oreo picture",
    }
  }
/>
`;
